example/ios/Example.xcodeproj/project.pbxproj

```diff
@@ -303,7 +303,7 @@
 					"FB_SONARKIT_ENABLED=1",
 				);
 				INFOPLIST_FILE = example/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.1;
```

Comment:
Interesting. Any particular reason? The native SDK still supports 10.

Reply:
I was running into a build error that I ended up fixing by updating the Xcode version instead. May revert this.

Reply:
> You didn't have to do anything with the `RadarRouteMode` declarations?

Not sure yet. Still trying to repro the original issue and then will make changes if needed.

Reply:
Fixed this by moving `RadarRouteMode` to a separate header and bumping versions.